awx/ui_next/src/screens/Credential/CredentialDetail/CredentialDetail.jsx

```diff
@@ -142,16 +224,26 @@ function CredentialDetail({ i18n, credential }) {
         {fields.map(field => renderDetail(field))}
 
         <UserDateDetail
+          id="credential-created-detail"
           label={i18n._(t`Created`)}
           date={created}
           user={created_by}
         />
         <UserDateDetail
+          id="credential-last_modified-detail"
           label={i18n._(t`Last Modified`)}
           date={modified}
           user={modified_by}
         />
       </DetailList>
+      {Object.keys(inputSources).length > 0 && (
+        <PluginFieldText>
+          <Trans>
```

Comment:
Why use `<Trans>` rather than doing 
```{i18n._(t * This field will be retrieved from an external secret management system using the specified credential.)```
? which has been our pattern to date

Reply:
Oh I thought we had decided to use Trans when the string stands alone in the DOM (ie not in an attribute and doesn't live outside the render function).  I'll change this.  There's no real benefit to one over the other in this case I don't think.